tests/MyGame/Example/Any.go

```diff
@@ -2,13 +2,49 @@
 
 package Example
 
-import "strconv"
+import (
+	"strconv"
+
+	flatbuffers "github.com/google/flatbuffers/go"
```

Comment:
How did we gain this import? why is it necessary when previously it wasn't

Reply:
The refactor for the Union Pack/UnPack methods requires access to flatbuffers.(Builder/Table) and so we need to import it

Reply:
The existing code already required access to these types?

Reply:
Yes

Reply:
So why didn't the existing code need this import?

Reply:
This code was in a different file. Since I have refactored the code into a different file it now requires the import in this file.